system-contracts/contracts/L1Messenger.sol

```diff
@@ -255,7 +255,7 @@ contract L1Messenger is IL1Messenger, ISystemContract {
         /// Check bytecodes
         uint32 numberOfBytecodes = uint32(bytes4(_totalL2ToL1PubdataAndStateDiffs[calldataPtr:calldataPtr + 4]));
         calldataPtr += 4;
-        bytes32 reconstructedChainedL1BytecodesRevealDataHash;
+        bytes32 reconstructedChainedL1BytecodesRevealDataHash = bytes32(0);
```

Comment:
Why?

Reply:
for all these OZ suggested adding in the default values. i dont have a strong opinion on keeping or removing